src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/HealthHeartbeatDefaultPayload.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    internal class HealthHeartbeatDefaultPayload : IHealthHeartbeatPayloadExtension
+    {
+        public const string FieldRuntimeFrameworkVer = "runtimeFramework";
+        public const string FieldTargetFramework = "targetFramework";
+        public const string FieldAppInsightsSdkVer = "appinsightsSdkVer";
+
+        public string EnabledProperties;
+
+        public HealthHeartbeatDefaultPayload(string enabledProperties)
+        {
+            this.EnabledProperties = enabledProperties;
+        }
+
+        public string Name => "SDKHealthHeartbeat";
+
+        public int CurrentUnhealthyCount
+        {
+            get { return 0; }
+        }
+
+        public IEnumerable<KeyValuePair<string, object>> GetPayloadProperties()
+        {
+            var payload = new Dictionary<string, object>();
+            if (this.IsFieldEnabled(FieldTargetFramework))
+            {
+                payload.Add(FieldTargetFramework, this.GetTargetFrameworkVer());
+            }
+
+            if (this.IsFieldEnabled(FieldAppInsightsSdkVer))
+            {
+                payload.Add(FieldAppInsightsSdkVer, this.GetAppInsightsSdkVer());
+            }
+
+            if (this.IsFieldEnabled(FieldRuntimeFrameworkVer))
+            {
+                payload.Add(FieldRuntimeFrameworkVer, this.GetRuntimeFrameworkVer());
+            }
+
+            return payload;
+        }
+
+        private bool IsFieldEnabled(string fieldName)
+        {
+            if (!this.EnabledProperties.Equals("*", StringComparison.OrdinalIgnoreCase))
+            {
+                return this.EnabledProperties.IndexOf(fieldName, StringComparison.OrdinalIgnoreCase) >= 0;
+            }
+
+            return true;
+        }
+
+        private string GetTargetFrameworkVer()
```

Comment:
this is not a target framework of the project, This is the framework that was picked by VS. Meaning that `netstandard1.3` will be used by `netstandard2.0` target framework. 

I wonder if there is a way to find out actual target framework of the project? Maybe analyzing some system libraries.